### DISCUSSION TITLE
Maybe Prepend Taiga styles in angular.json for variables overriding?

### DISCUSSION THREAD

**Discussion by Teebo** [taiga-family/taiga-ui#1039] [Ideas]
@waterplea would it be bad to do 

`targetOptions.styles = [...taigaStyles, ...styles];`

[here](https://github.com/TinkoffCreditSystems/taiga-ui/blob/c32b4247002e7f6b2693f1c727d6c7b3a2aa98aa/projects/taiga-schematics/schematics/ng-add/steps/add-taiga-styles.ts#L32)?

**Comment by waterplea** [comment]
Sounds reasonable. Can you drop a comment here?
https://github.com/TinkoffCreditSystems/taiga-ui/issues/1014
